### ISSUE TITLE
builder breaking snow unnecessarily

### ISSUE THREAD

**Issue by Spongman** [ldtteam/minecolonies#6344]
- Minecraft Version: 1.15.2
- MineColonies Version: minecolonies-0.13.548-ALPHA-universal
- Structurize Version: structurize-0.13.105-ALPHA-universal

Builders are breaking fallen snow in places where the schematic has air blocks. This is unnecessary.

**Comment by Raycoms** [comment]
air blocks are air blocks and not placeholder blocks though. So this is intentional I'd say.

**Comment by Spongman** [comment]
this makes no sense. there's no reason to break a partial snow block if it appears in an air block - that air block will get snow again (presumably) once the build is complete, so there's no reason to remove it.

also, in a snowy biome builders never complete builds because the snow is constantly replaced.

**Comment by ravenbuilder934** [comment]
If you'd like to reopen this, instead please create a new issue **filling out the template.**

**Comment by Spongman** [comment]
i filled out as much information as i could,

**Comment by Spongman** [comment]
reparing a building in order to fix a single broken block first requires every single piece of snow on the exposed upper surface to first be cleared. insane.
